lib/pages/App.js

```diff
@@ -141,11 +159,18 @@ export default class App extends Component {
         );
     } else if (page === pages.MAIN) {
       const viewDataLink = _.get(this.props, ['blipUrls', 'viewDataLink'], '');
-      let timezoneDropdown = this.renderTimezoneDropdown();
+      let timezoneDropdown = null;
+      let viewDataLinkButton = this.renderUploadListDoneButton();
+      let clinicUserBlock = this.renderClinicUserBlock();
```

Comment:
This (line 164) is repeated below in line 192 (and perhaps more places). We can consolidate, can't we?